var/spack/repos/builtin/packages/robodoc/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Robodoc(AutotoolsPackage):
+    """ROBODoc is program documentation tool."""
+
+    homepage = "https://www.xs4all.nl/~rfsber/Robo/index.html"
+    url      = "https://github.com/gumpu/ROBODoc/archive/refs/tags/v4.99.44.tar.gz"
+
+    maintainers = ['wscullin']
+
+    version('develop', git='https://github.com/gumpu/ROBODoc.git')
```

Comment:
I would add `git = ...` above where you define the url. Then you can use `branch='release'` here. This way, if someone ever adds master, you don't need to duplicate the git.